cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/StreamLocationNode.java

```diff
@@ -68,7 +69,7 @@ public void setStreamNode(StreamNode streamNode) {
     }
 
     //helper methods to extract from stream node
-
+    @JsonIgnore
```

Comment:
Why are these needed?

Reply:
They were just to helpers to reduce code on calls

streamLoc.getStreamLocationNode().getStreamNode().getStreamId()

vs

streamLoc.getStreamId()

Reply:
Right. Added these onto the codegen